tvb/datatypes/projections.py

```diff
@@ -78,7 +124,17 @@ class ProjectionRegionEEG(framework.ProjectionRegionEEGFramework,
         
     
     """
-    pass
+    @staticmethod
+    def from_file(source_file='projection_EEG_region.npy', instance=None):
```

Comment:
reading from bst .mat means not needing an intermediate step to generate the npy file which is slightly more trustworthy, but we can trust you this time ;)